pysqlsync/dialect/snowflake/object_types.py

```diff
@@ -49,23 +49,30 @@ def primary_key_constraint_id(self) -> LocalId:
 class SnowflakeColumn(Column):
     @property
     def default_expr(self) -> str:
+        default_expr = " DEFAULT "
+        if self.identity is True:
+            return ""
```

Comment:
If it's an identity (PK), should not have default. If not identity, but still doesn't have a deafult value, (probably?) a foreign key.